src/style.css

```diff
@@ -2,2 +1,71 @@
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.img {
+  display: block;
+}
```

Comment:
Avoid using tag names for styling like `.img`. If you need to apply a style to all `<img>` elements, it's better to use the `img` tag directly, unless there's a specific reason to use a class.